src/components/card/Card.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../styles/variables';
+
+.bdl-Card {
+    background-color: $white;
+    border-radius: 3px;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, .2);
+    margin: 0 auto;
```

Comment:
This will center the card in its container, which may not be desirable in many circumstances. Please consider multiple use cases for common components.

Reply:
I would not add margins at all, that depends/affects on the parent layout, component should ideally be encapsulated and full bleed